Resync `webtransport` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 5748618f58fdfdc830ab17b65d7bfa20498929e3
<pre>
Resync `webtransport` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=302346">https://bugs.webkit.org/show_bug.cgi?id=302346</a>
<a href="https://rdar.apple.com/164502176">rdar://164502176</a>

Reviewed by Brandon Stewart.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/3c78379a9191bc582cb55b10185d4863347df55a">https://github.com/web-platform-tests/wpt/commit/3c78379a9191bc582cb55b10185d4863347df55a</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/bidirectional-cancel-crash.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/close.https.any.js:
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/connect.https.any.js:
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/constructor.https.any.js:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.html.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.serviceworker-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.sharedworker-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.worker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/abort-stream-from-server.py:
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/echo.py:
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/echo_datagram_length.py:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/sendorder.py: Added.
(session_established):
(stream_data_received):
(stream_reset):
(datagram_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/server-connection-close.py:
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/server-read-then-close.py: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/server-connection-close.py.
(session_established):
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/token-count.py: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/echo.py.
(session_established):
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.js: Added.
(idl_array.self.webTransport.ready.catch):
(idl_array.self.webTransport.closed.catch):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.js: Added.
(validate_rtt_stats):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/w3c-import.log: Added.

Canonical link: <a href="https://commits.webkit.org/303369@main">https://commits.webkit.org/303369@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/67e85186d460852068a1b979bb4ce9d40a611f25

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/132152 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/4645 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/43176 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/139667 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/84072 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/960dc2d9-f3cd-4713-8492-0ec90471177a) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/4593 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/4406 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/101015 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/84072 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/5f694f2e-bb40-4dab-83d4-58e26b872037) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/135098 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/162/builds/3249 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/118368 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/81807 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/c75f1fb2-ed77-465a-921d-2941f2d3b6d9) 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/154/builds/3140 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac-debug~~](https://ews-build.webkit.org/#/builders/165/builds/1053 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/82886 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/111926 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/36492 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/142314 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/4314 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/37067 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/109390 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/4395 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/161/builds/3749 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/109566 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/27760 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/3273 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/114643 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/57560 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/4368 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/33032 "Passed tests") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/4199 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/4459 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/4327 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

stwrt reviewed (changes requested):
Several of the `webtransport` tests are failing. Can we update the TestExpecations.

macOS WK2 failures
```
imported/w3c/web-platform-tests/webtransport/idlharness.https.any.html
imported/w3c/web-platform-tests/webtransport/idlharness.https.any.serviceworker.html
imported/w3c/web-platform-tests/webtransport/idlharness.https.any.sharedworker.html
imported/w3c/web-platform-tests/webtransport/idlharness.https.any.worker.html
```

stwrt reviewed (approved):
This import LGTM if tests are passing.

Ahmad-S792 committed:
Resync `webtransport` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=302346
rdar://164502176

Reviewed by Brandon Stewart.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/3c78379a9191bc582cb55b10185d4863347df55a

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/bidirectional-cancel-crash.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/close.https.any.js:
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/connect.https.any.js:
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/constructor.https.any.js:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/datagram-bad-chunk.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.html.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.serviceworker-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.sharedworker-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.worker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/echo-large-bidirectional-streams.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/abort-stream-from-server.py:
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/echo.py:
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/echo_datagram_length.py:
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/sendorder.py: Added.
(session_established):
(stream_data_received):
(stream_reset):
(datagram_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/server-connection-close.py:
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/server-read-then-close.py: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/server-connection-close.py.
(session_established):
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/token-count.py: Copied from LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/echo.py.
(session_established):
(stream_data_received):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/handlers/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.js: Added.
(idl_array.self.webTransport.ready.catch):
(idl_array.self.webTransport.closed.catch):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/idlharness.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendorder.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.js: Added.
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/sendstream-bad-chunk.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.js: Added.
(validate_rtt_stats):
(promise_test.async t):
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.serviceworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.serviceworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.sharedworker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.sharedworker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.worker-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/stats.https.any.worker.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webtransport/w3c-import.log: Added.

Canonical link: https://commits.webkit.org/303369@main
